enhancements/coreos-encrypted-by-default.md

```diff
@@ -0,0 +1,422 @@
+---
+title: CoreOS Encrypted Disks By Default
+authors:
+  - "Ben Howard @behoward"
+reviewers:
```

Comment:
Should this have myself and the other folks tagged in?

Reply:
Done, please check the update.